doc/architecture/decisions/0013-use-s3.md

```diff
@@ -0,0 +1,39 @@
+# 13. Use AWS' S3 service for storing files
+
+Date: 2019-11-21
+
+## Status
+
+Accepted
+
+## Context
+
+There is a user need to store evidence in the form of images, videos, and PDF
+documents within the Report a Defect system.
+
+This is a common need across Hackney projects and S3 is already in use across
+the organisation. As such there is a project in flight to wrap S3 functionality
+in an API that services can use.
+
+At the time of writing, this API project is not ready nor is there an agreed
+timeframe for when it will be ready.
+
+There is no user need to be able to edit an image once uploaded, as such the S3
+buckets will not need to have versioning enabled.
+
+## Decision
+
+Since there is an established pattern of using S3 for asset storage, we have
+decided to continue this pattern and use S3 for this project.
+
+We will be requested two buckets are made, one for the staging environment and
```

Comment:
I'm a little confused at the phrasing here. "We will be requested", or "We will be requesting"?

Reply:
D'oh, thanks! Will address.

Reply:
@CristinaRO I've rewritten to use the correct tense